### PR TITLE
new goal files for Ubuntu 24.04

### PR DESCRIPTION
**Pull request by markro49**

### CODE REVIEW

**Comment by mernst** [comment]
I believe I need to make new Docker images to make these tests pass.  Is that right?

**Comment by markro49** [comment]
They ci checks should pass as the new goal files are not being used as yet. I ran the tests on a 24.04 wsl box.

File: tests/kvasir-tests/Makefile.common [hunk 1]
```diff
@@ -121,6 +121,9 @@ endif
 endif
 
 ifeq ($(OS_ID),Ubuntu)
+ifeq ($(OS_REL),24.04)
+	GOAL := goal-Ubuntu2404
+endif
 ifeq ($(OS_REL),23.10)
```
Comment (mernst) [inline tests/kvasir-tests/Makefile.common:127]: Should this and some other `ifeq` statements be removed, since the test for "24.04" now exists?

**Review by mernst — APPROVED** [review]
Thanks!

File: tests/kvasir-tests/Makefile.common (hunk 1 above)
Comment (markro49) [reply tests/kvasir-tests/Makefile.common:127]: A few goal files differ between Ubuntu 22.04 and 24.04; so I think we need to keep this change for now.  We can review again when we decide what to do about the other, stale goal files.

**Review by mernst — APPROVED** [review]
Thanks!

**Pull request merged**